.circleci/config.yml

```diff
@@ -187,6 +187,22 @@ jobs:
               --dotenv-path .env \
               --justfile ../../../single.just \
               simulate
+  just_simulate_sep-op-006-0:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - run:
+          name: just simulate sep/006-0-sc-guardian-updates
```

Comment:
The changes of this and the subsequent runbook are not restricted to the guardian, but they are necessary to enable the "guardian" changes:

1. Increase Threshold: Properly set up Security Council by landing Liveness Module and increase threshold to 75%.
2. Reduce Pause: Reduce non-SC's ability to pause indefinitely.

Unless it's important to you, I'd like to keep that "slug" the same for now.

Note that this ceremony _does not_ make threshold changes, because we've decided that both of the Sepolia Safes [should have the same threshold](https://www.notion.so/oplabs/Proposal-Align-Sepolia-roles-config-with-Mainnet-a1fb54672f514a5aaec65a564671c621?pvs=4).

I called this out in the [README](https://github.com/ethereum-optimism/superchain-ops/pull/165/files#diff-0b09b53c8e385ea409a78cd6e08690df2ba36270cdf3cf12f0e1fb9b36fbeb30R12) as well.